doc/2/api/controllers/server/capabilities/index.md

```diff
@@ -0,0 +1,73 @@
+---
+code: true
+type: page
+title: capabilities
+---
+
+# Capabilities
+
+
+
+Returns the current Kuzzle capabilities.
+
+This route should only be accessible to administrators, as it might return sensitive information about the backend.
```

Comment:
I don't agree, this action (not route) should be accessible to most of the users. As mentioned in the original issue, it can allows SDK to behave accordingly automatically.

This action purpose is to inform the client about what kind of limits or behavior it should expect from Kuzzle.

You can look at the original issues for more info about the usage and the expected response